### PR TITLE
Upgrade Claims

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Do we need `Statement` and `Preclaims`? @HackFisher

**Comment by HackFisher** [comment]
> Do we need [`Statement`](Statement) and [`Preclaims`](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/claims.rs#L185)? @HackFisher

We do not need this for now.

**Comment by AurevoirXavier** [comment]
I also made a PR for polkadot.

https://github.com/paritytech/polkadot/pull/1577

File: frame/claims/src/lib.rs [hunk 1]
```diff
@@ -227,6 +294,45 @@ decl_module! {
 				}
 			}
 		}
+
+		#[weight = (
+			T::DbWeight::get().reads_writes(4, 4) + 100_000_000_000,
+			DispatchClass::Normal,
+			Pays::No
+		)]
+		fn move_claim(origin,
```
Comment (HackFisher) [inline frame/claims/src/lib.rs:303]: What is move claim?

Comment (HackFisher) [reply frame/claims/src/lib.rs:303]: Add some docs here.

**Comment by HackFisher** [comment]
I prefer

```
ensure!(Self::claims(address).is_none(), "Can NOT Move Claim to an EXISTED Address");

```

**Comment by AurevoirXavier** [comment]
Merge into #232

**Pull request closed without merging**